This is failing like this:

```
/usr/bin/gfortran -g -funroll-loops  -fallow-argument-mismatch -O3 CMakeFiles/tocgrib.dir/makwmo.F90.o CMakeFiles/tocgrib.dir/mkfldsep.F90.o CMakeFiles/tocgrib.dir/tocgrib.F90.o -o tocgrib  /home/runner/bacio/lib/libbacio.a /home/runner/w3emc/lib/libw3emc_4.a /home/runner/bacio/lib/libbacio.a
/usr/bin/ld: CMakeFiles/tocgrib.dir/tocgrib.F90.o: in function `MAIN__':
/home/runner/work/NCEPLIBS-g2/NCEPLIBS-g2/g2/utils/tocgrib.F90:73: undefined reference to `w3as00_'
collect2: error: ld returned 1 exit status
make[2]: *** [utils/CMakeFiles/tocgrib.dir/build.make:129: utils/tocgrib] Error 1
make[2]: Leaving directory '/home/runner/work/NCEPLIBS-g2/NCEPLIBS-g2/g2/build'
make[1]: *** [CMakeFiles/Makefile2:1536: utils/CMakeFiles/tocgrib.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....

```
But this PR contains only documentation changes...

--- a/.github/workflows/Linux_options.yml
+++ b/.github/workflows/Linux_options.yml
@@ -88,7 +88,7 @@ jobs:
       with:
         repository: NOAA-EMC/NCEPLIBS-w3emc
         path: w3emc
-        ref: v${{ matrix.w3emc-version }}
+        ref: v${{ matrix.w3emc-version }}-1
 
     - name: build-w3emc
       if: steps.cache-w3emc.outputs.cache-hit != 'true'

--- a/.github/workflows/Linux_options.yml
+++ b/.github/workflows/Linux_options.yml
@@ -96,7 +96,7 @@ jobs:
         cd w3emc
         mkdir build
         cd build
-        cmake -DCMAKE_PREFIX_PATH=~/bacio -DCMAKE_INSTALL_PREFIX=~/w3emc -DBUILD_WITH_BUFR=OFF -DBUILD_WITH_EXTRA_DEPS=ON ..
+        cmake -DCMAKE_PREFIX_PATH=~/bacio -DCMAKE_INSTALL_PREFIX=~/w3emc -DBUILD_WITH_BUFR=OFF -DBUILD_WITH_EXTRA_DEPS=ON -DBUILD_DEPRECATED=ON ..
         make -j2
         make install
         

--- a/src/params.F90
+++ b/src/params.F90
@@ -269,6 +269,13 @@ module params
   data paramlist(235) /gribparam(2, 189, 0, 0, 15, 'VPTMP')/
   data paramlist(236) /gribparam(2, 198, 0, 1, 207, 'NCIP')/
   data paramlist(237) /gribparam(2, 239, 0, 1, 208, 'SNOT')/
+  ! The following param shares the same GRIB2 discipline, category,
+  ! and number as PRMSL. Since the parameter lookup subroutine starts
+  ! at the beginning of the list, and returns the first abbreviation
+  ! with matching G2 numbers, the MSLSA abbreviation can never be
+  ! returned. According to Boi: In the past, some user used PRMSL or
+  ! MSLSA in grib1. Then, they converted to grib2 and these parameters
+  ! were in same "0,3,1".
   data paramlist(238) /gribparam(2, 128, 0, 3, 1, 'MSLSA')/
   data paramlist(239) /gribparam(2, 137, 0, 3, 199, 'TSLSA')/
   data paramlist(240) /gribparam(129, 141, 0, 3, 200, 'PLPL')/

